Dictionaries/hrv_OCRFixReplaceList.xml

```diff
@@ -861,7 +864,7 @@
     <RegEx find="\b([nN])eg([aeu]|om|ama)\b" replaceWith="$1jeg$2" />
     <RegEx find="\b([nN])ež(an|n[aeiou]|nom|nima)\b" replaceWith="$1jež$2" />
     <RegEx find="([nN])emačk" replaceWith="$1jemačk" />
-    <RegEx find="([nN])j?emci" replaceWith="Nijemci" />
+    <RegEx find="([nN])j?em(a?)c([ei])" replaceWith="Nijem$1c$2" />
```

Comment:
maybe it's better like this?:
` <RegEx find="[nN]j?em(a?)c([ei])" replaceWith="Nijem$1c$2" />`

whether it's n or N it always must be N, since it's name for people who live in Germany.